<launch>
    <node pkg="visual_ekf" type="visual_ekf_test" name="propagate_test" output="screen"/>
    <node pkg="visual_ekf" type="visual_ekf_node" name="visual_ekf_node" output="screen">
        <param name="angular_fused" type="string" value="/visual_ekf/test_twist"/>
        <param name="visual_topic" type="string" value="/pnp_twist"/>
        <param name="publisher_topic" type="string" value="/visual_ekf/shield_pose_fused"/>
        <param name="gyroscope_noise_weight" type="double" value="10.0"/>
        <param name="visual_pose_weight" type="double" value="10000.0"/>
        <param name="node_sleep_time" type="int" value="10"/>
    </node>

</launch>
